### ISSUE TITLE
Issue with "@graphql-eslint/graphql" processor

### ISSUE THREAD

**Issue by tnrich** [B2o5T/graphql-eslint#1038]
### Issue workflow progress

_Progress of the issue based on the [Contributor Workflow](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md#a-typical-contributor-workflow)_

- [ ] 1. The issue provides a reproduction available on GitHub, Stackblitz or CodeSandbox
> Please make sure the graphql-eslint version under `package.json` matches yours.
- [ ] 2. A failing test has been provided
- [ ] 3. A local solution has been provided
- [ ] 4. A pull request is pending review

---

**Describe the bug**

**To Reproduce**
- Checkout this branch: https://github.com/TeselaGen/teselagen-react-components/tree/tnrDemonstrateLintingWeirdness
- yarn (at the top level)
- open up the two files like so (src/FormComponents/index.js and .eslintrc.js): 

https://user-images.githubusercontent.com/2730609/164784519-b25e6258-a7c2-4487-b4e4-679a3ad86c8b.mov

See how the normal eslint highlighting gets really messed up (in vscode)

**Expected behavior**

The eslint error highlighting should NOT change with the addition of this code: 
```
module.exports = {
  overrides: [
    {
      files: ["*.js"],
      processor: "@graphql-eslint/graphql",
      plugins: ["@graphql-eslint"],
      parserOptions: {
        schema: "../../server/src/graphql-lims/schema/schema.gql",
      },
    },
  ],
};
```

**Environment:**

- OS:
- `@graphql-eslint/eslint-plugin`:
- Node.js:

**Additional context**

**Comment by B2o5T** [comment]
You have an invalid configuration of ESLint
Add the following to fix your error output
```js
  env: {
    browser: true,
    es2022: true
  },
  parserOptions: {
    sourceType: "module",
    ecmaFeatures: {
      jsx: true
    }
  },
```
I'm closing it since there is nothing wrong with `graphql-eslint`

**Comment by tnrich** [comment]
Hey @B2o5T thanks for you quick response. Unfortunately I'm still seeing this issue after updating the eslint config as suggested: 


https://user-images.githubusercontent.com/2730609/164822232-bde2512c-4a53-478c-81e1-d471cf1f16aa.mov

I suggest re-opening. I've pushed the updates to https://github.com/TeselaGen/teselagen-react-components/tree/tnrDemonstrateLintingWeirdness

**Comment by tgreen7** [comment]
here is a minimal repo with the same issue https://github.com/tgreen7/graphql-eslint-test

<img width="652" alt="Screen Shot 2022-04-22 at 4 53 04 PM" src="https://user-images.githubusercontent.com/10415101/164827606-d1d77f67-eb88-4153-baf2-ed9e4c1fe9e4.png">

**Comment by B2o5T** [comment]
@tnrich @tgreen7 thank you guys, can confirm that `graphql-eslint` had an issue with the processor, which I fixed in [`3.10.3`](https://github.com/B2o5T/graphql-eslint/releases/tag/%40graphql-eslint%2Feslint-plugin%403.10.3) version.
Try it!
